assets/inbuilt/transformers/springboot/java-springboot/mappings/java2images_tags.json

```diff
@@ -0,0 +1 @@
, "created": "2019-02-13T12:04:25.542347612Z", "architecture": "amd64"}, {"name": "jboss/wildfly:9.0.1.Final", "created": "2019-02-13T13:24:14.978433221Z", "architecture": "amd64"}, {"name": "jboss/wildfly:14.0.0.Final", "created": "2019-02-13T12:15:12.523514602Z", "architecture": "amd64"}, {"name": "jboss/wildfly:13.0.0.Final", "created": "2019-02-13T11:50:10.227797018Z", "architecture": "amd64"}, {"name": "jboss/wildfly:11.0.0.Final", "created": "2019-02-13T11:38:50.496013193Z", "architecture": "amd64"}, {"name": "jboss/wildfly:10.1.0.Final", "created": "2019-02-13T13:16:47.18222971Z", "architecture": "amd64"}, {"name": "jboss/wildfly:10.0.0.Final", "created": "2019-02-13T11:54:56.216677268Z", "architecture": "amd64"}]}, {"version_id": "1.7", "images": [{"name": "jboss/wildfly:8.1.0.Final", "created": "2019-02-13T13:06:29.897019728Z", "architecture": "amd64"}, {"name": "jboss/wildfly:8.2.0.Final", "created": "2019-02-13T11:21:46.257921524Z", "architecture": "amd64"}]}, {"version_id": "11", "images": [{"name": "jboss/wildfly:23.0.0.Final", "created": "2021-03-16T12:20:11.958482488Z", "architecture": "amd64"}, {"name": "jboss/wildfly:22.0.0.Final", "created": "2021-01-28T11:54:19.11377621Z", "architecture": "amd64"}, {"name": "jboss/wildfly:21.0.2.Final", "created": "2021-01-05T15:49:37.272274369Z", "architecture": "amd64"}, {"name": "jboss/wildfly:15.0.1.Final", "created": "2019-02-13T11:29:49.641638018Z", "architecture": "amd64"}, {"name": "jboss/wildfly:17.0.1.Final", "created": "2019-07-10T07:05:52.428610517Z", "architecture": "amd64"}, {"name": "jboss/wildfly:18.0.1.Final", "created": "2019-11-18T08:31:01.841738515Z", "architecture": "amd64"}, {"name": "jboss/wildfly:20.0.0.Final", "created": "2020-06-22T09:42:27.288648004Z", "architecture": "amd64"}, {"name": "jboss/wildfly:21.0.0.Final", "created": "2020-10-21T09:25:31.897267464Z", "architecture": "amd64"}, {"name": "jboss/wildfly:22.0.1.Final", "created": "2021-02-12T08:21:48.823067126Z", "architecture": "amd64"}, {"name": "jboss/wildfly:15.0.0.Final", "created": "2019-02-13T13:53:43.861482142Z", "architecture": "amd64"}, {"name": "jboss/wildfly:23.0.1.Final", "created": "2021-04-15T07:42:10.113693208Z", "architecture": "amd64"}, {"name": "jboss/wildfly:21.0.1.Final", "created": "2020-12-02T12:57:04.468022499Z", "architecture": "amd64"}, {"name": "jboss/wildfly:19.0.0.Final", "created": "2020-03-27T10:08:38.660818401Z", "architecture": "amd64"}, {"name": "jboss/wildfly:16.0.0.Final", "created": "2020-09-07T09:21:27.707384617Z", "architecture": "amd64"}, {"name": "jboss/wildfly:19.1.0.Final", "created": "2020-05-31T19:09:37.883038006Z", "architecture": "amd64"}, {"name": "jboss/wildfly:17.0.0.Final", "created": "2019-06-11T07:37:44.87732309Z", "architecture": "amd64"}, {"name": "jboss/wildfly:20.0.1.Final", "created": "2020-09-07T09:25:05.065383866Z", "architecture": "amd64"}, {"name": "jboss/wildfly:18.0.0.Final", "created": "2019-10-07T08:07:41.180078006Z", "architecture": "amd64"}]}]}]}
```

Comment:
The structure of this json seems to be custom made for this very specific implementation. If we have to say add more filters other than java version or app server, we need another json. Would it be better if we flatten it out, and then have a list of images and properties for each image, which will include app server name, java version and more. This way, we can add any more meta data to the images without having to worry about the hierarchy.